Makefile

```diff
@@ -98,5 +100,12 @@ test-rc-deps: ## Test providers RC by building an image with given dependencies
 shell:  ## Runs a shell within a container (Allows interactive session)
 	docker compose -f dev/docker-compose.yaml run --rm airflow-scheduler bash
 
+bump-version:  ## Bump versions in files locally. By default bump to DEV. set env ASTRO_PROVIDER_VERSION to change default value (dev | patch | major | minor).
+	@if [ $(ASTRO_PROVIDER_VERSION) = "dev" ]; then\
+		cz bump --version-type semver --increment minor --devrelease 1 --files-only;\
+	else \
+	  cz bump --version-type semver --files-only --increment $(ASTRO_PROVIDER_VERSION);\
```

Comment:
What happens when the previous release was 1.15.6, next we bump it to 1.16.0-dev1 and then want to create a patch release. Will it be able to be set to 1.15.7 from 1.16.0-dev1?

Reply:
good point, I think `commitizen` don't know the last released version so should be 1.16.1.

Reply:
but let me check if there is way to fix it

Reply:
I don't think they can move backwards, but one can pass the manual version for example `cz bump 1.15.6` maybe I'll few more condition for this, thanks for pointing out

Reply:
In essence, commitizen handles project versioning in accordance with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and [SemVer](https://semver.org/). Thus, downgrading project version might not be something that has ever considered by commitizen